content/04-conclusion.md

```diff
@@ -1,10 +1,8 @@
 ## Conclusion
 
 nf-core/eager is an efficient, portable, and accessible pipeline for processing
-ancient DNA genomic data. The re-implementation with nextflow and nf-core
+ancient DNA genomic data. The re-implementation with Nextflow and nf-core
```

Comment:
Are we going upper case o lower case? Above you put it lower case?

Reply:
Nextflow it should be across the entire document. nf-core too :-)